pipenv/cmdparse.py

```diff
@@ -2,31 +2,61 @@
 import re
 import shlex
 
+import tomlkit
```

Comment:
This should be `from pipenv.vendor import tomlkit` to ensure that imports will work for all users -- its probably that tomlkit was just installed in your environment for this to work, but we have it vendor'd in.

Reply:
ok cool. there were some other places not using the vendored version and i updated those as well